spec.json

```diff
@@ -344,6 +346,7 @@
         },
         "largeMotor": {
             "friendlyName": "Large Motor",
+            "systemDeviceNameConvention": "motor{0}",
```

Comment:
I agree -- we should normalize on a way of specifying this information and stick with it. I'll need to take a look at the underlying device code in my JS lib to refresh my memory on how I do it, but I _think_ I currently transform it into a regex. That  would probably be a good format to use.